src/main/java/org/jboss/tm/TransactionLocalDelegateImpl.java

```diff
@@ -21,185 +21,159 @@
-         return null;
-      return sync.getValue(local);
-   }
-
-   public void storeValue(TransactionLocal local, Transaction tx, Object value)
-   {
-      TransactionLocalSynchronization sync = getSynchronization(tx, true);
-      sync.setValue(local, value);
-   }
-
-   public boolean containsValue(TransactionLocal local, Transaction tx)
-   {
-      TransactionLocalSynchronization sync = getSynchronization(tx, false);
-      if (sync == null)
-         return false;
-      return sync.containsValue(local);
-   }
-
-   // InnerClasses ---------------------------------------------------
-
-   protected static class TransactionLocalSynchronization
-      implements Synchronization
-   {
-      protected Transaction tx;
-
-      private Map<Object, Object> valuesByLocal = Collections.synchronizedMap(new HashMap<Object, Object>());
-
-      protected ReentrantLock reentrantLock = new ReentrantLock();
-
-      public TransactionLocalSynchronization(Transaction tx)
-      {
-         this.tx = tx;
-      }
-
-      public void beforeCompletion()
-      {
-      }
-
-      public void afterCompletion(int status)
-      {
-         removeSynchronization(tx);
-         valuesByLocal.clear(); // Help the GC
-      }
-
-      public void lock(Object local) throws InterruptedException
-      {
-         boolean locked = reentrantLock.tryLock(60000, TimeUnit.MILLISECONDS);
-         if (locked == false)
-            throw new IllegalStateException("Failed to acquire lock within 60 seconds.");
-      }
-      
-      public void unlock(Object local)
-      {
-         reentrantLock.unlock();
-      }
-
-      public Object getValue(Object local)
-      {
-         return valuesByLocal.get(local);
-      }
-
-      public void setValue(Object local, Object value)
-      {
-         valuesByLocal.put(local, value);
-      }
-
-      public boolean containsValue(Object local)
-      {
-         return valuesByLocal.containsKey(local);
-      }
-   }
+            return result;
+        }
+    }
+
+    /**
+     * Remove a synchronization
+     *
+     * @param tx the transaction to remove
+     */
+    protected static void removeSynchronization(Transaction tx) {
+        synchronizationsByTransaction.remove(tx);
+    }
+
+    // Constructor ---------------------------------------------------
+
+    /**
+     * Construct a new delegate for the given transaction manager
+     *
+     * @param manager the transaction manager
+     */
+    public TransactionLocalDelegateImpl(TransactionManager manager) {
+        this.manager = manager;
+    }
+
+    public void lock(TransactionLocal local, Transaction tx) throws InterruptedException {
+        TransactionLocalSynchronization sync = getSynchronization(tx, true);
+        sync.lock(local);
+    }
+
+    public void unlock(TransactionLocal local, Transaction tx) {
+        TransactionLocalSynchronization sync = getSynchronization(tx, false);
+        if (sync != null) sync.unlock(local);
```

Comment:
This looks wrong, one half of the flow control is on its own line whilst the other half is on a line by itself.
Was this your addition or did you use a tool?

Can you check all of the code for where this style is used and modify it so that it is consistent with normal styling rules.

Reply:
> This looks wrong, one half of the `if-then-else` statement is collapsed with the `if` whilst the other half, the else arm, is on a line by itself. Was this your addition or did you use a tool?
> 
> Can you check all of the code for where this style is used and modify it so that it is consistent with normal styling rules.

I agree with it, I reformatted it to original where ever required.